pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -974,8 +974,14 @@ class FunctionToolCallEvent:
     """Event type identifier, used as a discriminator."""
 
     @property
+    def tool_call_id(self) -> str:
+        """An ID used for matching details about the call to its result."""
+        return self.part.tool_call_id
+
+    @property
+    @deprecated('`call_id` is deprecated, use `tool_call_id` instead.')
     def call_id(self) -> str:
-        """An ID used for matching details about the call to its result. If present, defaults to the part's tool_call_id."""
+        """An ID used for matching details about the call to its result."""
         return self.part.tool_call_id
```

Comment:
@proever The coverage test is complaining about like 988, but I think it actually means this line, so let's see if this does it:

```suggestion
        return self.part.tool_call_id  # pragma: no cover
```

Reply:
@proever That didn't fix it yet :/ The line number it's complaining about is definitely wrong, but assuming there is in fact an uncovered line, can you reason about which one it may be and see if adding a pragma there does it?

I'll make a note to look at the wrong line numbers